doc/run/k8s/install-sqlflow.yaml

```diff
@@ -1,59 +1,49 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
   name: sqlflow-server
 spec:
-  selector:
-    matchLabels:
-      app: sqlflow-server
-  template:
-    metadata:
-      labels:
-        app: sqlflow-server
-    spec:
-      containers:
-      - name: server
-        image: sqlflow/sqlflow
-        imagePullPolicy: Always
-        env:
-        - name: SQLFLOW_WORKFLOW_LOGVIEW_ENDPOINT
-          value: "http://localhost:8001"
-        - name: SQLFLOW_WORKFLOW_STEP_IMAGE
-          value: "sqlflow/sqlflow
+  containers:
+  - name: server
+    image: sqlflow/sqlflow
+    env:
+    - name: SQLFLOW_WORKFLOW_LOGVIEW_ENDPOINT
+      value: "http://localhost:8001"
+    - name: SQLFLOW_WORKFLOW_STEP_IMAGE
+      value: sqlflow:server
+    command:
+      - sqlflowserver
+      - --argo-mode
+  - name: notebook
+    image: sqlflow/sqlflow
+    ports:
+    - containerPort: 8888
+      name: notebook
+    command:
+    - bash
+    - -c
+    - 'export SQLFLOW_DATASOURCE=mysql://root:root@tcp\(${MY_POD_IP}:3306\)/?maxAllowedPacket=0 && jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root'
```

Comment:
This line seems that we run the Jupyter and the SQLFlow server in the same container. Should we separate the Jupyter server out as we did with the MySQL server?

Reply:
That's a good idea, the **all-in-one** Docker image is not needed.